pymeasure/instruments/common_base.py

```diff
@@ -436,6 +445,11 @@ class SpecificInstrument(GenericInstrument):
                  FutureWarning)
             values_kwargs.update(kwargs)
 
+        if command_process is None:
+            command_process = null_operation
```

Comment:
why not keep  `lambda c: c` here? Why do we need an (otherwise useless) externally defined method here?

Reply:
flake does not like assigning lambda functions.
Should I use `noqa`?

Reply:
ah, yeah, I did not think of that. Yeah, noqa is fine here.